frontend/src/models/Question.model.tsx

```diff
@@ -7,13 +7,16 @@ export class Question {
   id: number;
   _id: string;
 
+
   constructor(
     _id: string,
     id: number,
     title: string,
     descMd: string,
     categories: string[],
-    difficulty: number
+    difficulty: number,
+    solved: boolean = false,
+    solvedDate?: string
   ) {
     this.title = title;
     this.descMd = descMd;
```

Comment:
remove from constructor

Reply:
Fixed this in [Minor changes to Question model](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g10/pull/43/commits/87f847730550b8ddf472a3ac80e3597ec32ac896)